@@@ Configure php_fpm to use a larger, fixed number of threads
--- debian/etc/php/7.3/fpm/pool.d/www.conf
+++ permanent/etc/php/7.3/fpm/pool.d/www.conf
@@ -100,7 +100,7 @@
 ;             pm.process_idle_timeout   - The number of seconds after which
 ;                                         an idle process will be killed.
 ; Note: This value is mandatory.
-pm = dynamic
+pm = static

 ; The number of child processes to be created when pm is set to 'static' and the
 ; maximum number of child processes when pm is set to 'dynamic' or 'ondemand'.
@@ -111,7 +111,7 @@
 ; forget to tweak pm.* to fit your needs.
 ; Note: Used when pm is set to 'static', 'dynamic' or 'ondemand'
 ; Note: This value is mandatory.
-pm.max_children = 5
+pm.max_children = 360

 ; The number of child processes created on startup.
 ; Note: Used only when pm is set to 'dynamic'
